src/components/Quests/Rewards/RewardsCarousel.tsx

```diff
@@ -0,0 +1,147 @@
+import { ChainId } from '@lifi/types';
+import OpenInNewIcon from '@mui/icons-material/OpenInNew';
+import { Box, Typography } from '@mui/material';
+import { Button } from 'src/components/Button';
+import {
+  useAccount,
+  useSwitchChain,
+  useWaitForTransactionReceipt,
+  useWriteContract,
+} from 'wagmi';
+import { MerklDistribABI } from '../../../const/abi/merklABI';
+import { FlexCenterRowBox } from '../QuestPage/QuestsMissionPage.style';
+import { RewardsAmountBox } from './RewardsAmountBox/RewardsAmountBox';
+import {
+  ClaimButtonBox,
+  EarnedTypography,
+  RewardsCarouselContainer,
+  RewardsCarouselMainBox,
+  RewardsOpenIconButton,
+} from './RewardsCarousel.style';
+
+interface RewardsCarouselProps {
+  hideComponent: boolean;
+  rewardAmount: number;
+  accumulatedAmountForContractBN: string;
+  isMerklSuccess: boolean;
+  proof: string[];
+}
+
+const CLAIMING_CONTRACT_ADDRESS = '0x3Ef3D8bA38EBe18DB133cEc108f4D14CE00Dd9Ae';
```

Comment:
Might need to be dynamic isn't it?

Reply:
Yes done in the next PR: https://github.com/jumperexchange/jumper-exchange/pull/1268